src/main/src/test/java/org/vfny/geoserver/crs/OvverideTransformationsTest.java

```diff
@@ -27,14 +27,29 @@
 
 public class OvverideTransformationsTest extends GeoServerSystemTestSupport {
     
+    private static final String JAVA_IO_TMPDIR = "java.io.tmpdir";
     private static final String SOURCE_CRS = "EPSG:TEST1";
     private static final String TARGET_CRS = "EPSG:TEST2";
     
     private static final double[] SRC_TEST_POINT = {39.592654167, 3.084896111};
     private static final double[] DST_TEST_POINT = {39.594235744481225, 3.0844689951999427};
+    private static String OLD_TMP_VALUE;
+    
+    
+    @AfterClass
+    public static void clearTemp() {
+        if(OLD_TMP_VALUE == null) { 
+            System.clearProperty(JAVA_IO_TMPDIR);
+        } else {
+            System.setProperty(JAVA_IO_TMPDIR, OLD_TMP_VALUE);
+        }
+    }
 
     @Override
     protected void onSetUp(SystemTestData testData) throws Exception {
+        OLD_TMP_VALUE = System.getProperty(JAVA_IO_TMPDIR);
+        System.setProperty(JAVA_IO_TMPDIR, new File("./target").getCanonicalPath());
```

Comment:
I do not like tests that change system properties. Why is this needed or expected to fix anything? Is this a Windows fix?